docs/preview-environments/drivers/os-env.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: os-env
+slug: /preview-environments/drivers/os-env
+title: os-env
+---
+
+This preview environments driver can be used to read environment variables that contain the prefix `PORTER_APPLY_`.
+
+For example, to read in all the environment variables that start with `PORTER_APPLY_`:
```

Comment:
Showing an example Github actions file that is modified to inject `PORTER_APPLY_` variables would also be helpful here.